CONTRIBUTING.md

```diff
@@ -52,16 +52,14 @@ make test TEST="Tests/Util/FilenameUtilsTest.php"
 
 will allow to tests single Test Classes.
 
-There are 3 environments available: PHP 7.3, 7.4 and 8.0.
+There are 2 environments available: PHP 7.4 and 8.0.
```

Comment:
This is a bit too much.
You need to try to workaround tests for PHP 7.3 before this

Reply:
Either adding 7.3 support for oneup/flysystem-bundle
Either removing this bundle on the 7.3 ci ?

Reply:
We can disable platform check for composer on 7.3 in tests. If something fails with flysystem-bundle, we can mark tests as skipped when PHP is below 7.4

Reply:
What about using a variable, as I just did?

Reply:
it could work